IMPORTANT_UPDATES.md

```diff
@@ -0,0 +1,14 @@
+## [10/20/2023] Migration from `TERRAFORM_OPEN_SOURCE` to `EXTERNAL` product type
+AWS Service Catalog plans on introducing some important changes to their support of Terraform Open Source starting October 12, 2023. If you are not already aware, HashiCorp announced a switch from the Mozilla Public License (MPL) 2.0 the Business Source License for Terraform. This change impacts customers who are using Terraform Open Source with Service Catalog, because references to 'open source' will be changed. Currently, Service Catalog references ‘open-source’ language in various artifacts such as our APIs, console, documentation, and a publicly available reference engine (TRE) that can be accessed through GitHub.
```

Comment:
Typo? - HashiCorp announced a switch from the Mozilla Public License (MPL) 2.0  [to] the Business Source License for Terraform

Reply:
Fixed with new commit